news/views.py

```diff
@@ -1,8 +1,11 @@
+import markdown
 from django.shortcuts import render
 
 from .models import NewsObject
 
 
 def news(request):
     news_objects = NewsObject.objects.all().order_by("-created_on")
-    return render(request, "portal/news.html", locals())
+    for news in news_objects:
+        news.body_html = markdown.markdown(news.body)
+    return render(request, "news/news.html", locals())
```

Comment:
I think we should avoid the `locals`  function and instead be explicit about the `context`. Perhaps
```suggestion
    for news in news_objects:
        news.body_html = markdown.markdown(news.body)
    return render(request, "news/news.html", {"news_objects": news_objects})
```

Reply:
good point! now did that.